### ISSUE TITLE
Exception: "The getter 'screencapStream' was called on null" when enabled is omitted

### ISSUE THREAD

**Issue by limonadev** [peiffer-innovations/automated_testing_framework#4]
## Steps to reproduce
1. Create some basic app with `TestRunner`
2. Don't set the `enabled` property.

## Example
```dart
import 'package:automated_testing_framework/automated_testing_framework.dart';
import 'package:flutter/material.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatefulWidget {
  @override
  _MyAppState createState() => _MyAppState();
}

class _MyAppState extends State<MyApp> {
  final GlobalKey<NavigatorState> _navigatorKey = GlobalKey<NavigatorState>();
  TestController _testController;

  Future<void> _onReset() async {
    setState(() {});
  }

  @override
  void initState() {
    super.initState();
    _testController = TestController(
      navigatorKey: _navigatorKey,
      onReset: _onReset,
    );
  }

  @override
  Widget build(BuildContext context) {
    return TestRunner(
      controller: _testController,
      child: MaterialApp(
        key: UniqueKey(),
        navigatorKey: _navigatorKey,
        title: 'Automated Testing Demo',
        theme: ThemeData(
          primarySwatch: Colors.blue,
          visualDensity: VisualDensity.adaptivePlatformDensity,
        ),
        home: Container(),
      ),
    );
  }
}
```

**Comment by jpeiffer** [comment]
What mode where you running in?  Asking because the app mode is actually important for how the TestRunner self-bootstraps.

**Comment by limonadev** [comment]
I was running on debug mode. But I was checking and I think the problem is happening here:
```
controller = enabled == true ? controller : null,
```
because if I don't pass the `enabled` property, it will be null so the controller will be also null.

**Comment by jpeiffer** [comment]
Thanks.  I'll take a look.  I'm in the middle of an update to the framework that I plan to check in today, so I'll be sure this use case is fixed as part of that.

**Comment by jpeiffer** [comment]
I believe this is fixed in 1.0.2.  Can you retest and confirm?

**Comment by limonadev** [comment]
It's fixed.
